readme.md

```diff
@@ -78,25 +78,6 @@ try {
 }
 ```
 
-### Cancelling a spawned process
-
```

Comment:
We should also remove the documentation [of the `cancel()` method](https://github.com/sindresorhus/execa/blob/main/readme.md#cancel).